cpp/rrtmgp_conversion.h

```diff
@@ -308,42 +377,25 @@ typename KView::non_const_value_type maxval(const KView& view)
 
 // Get sum of view
 template <typename KView>
-typename KView::non_const_value_type sum(const KView& view)
+std::conditional_t<std::is_same<typename KView::non_const_value_type, bool>::value, int, typename KView::non_const_value_type>
+sum(const KView& view)
 {
   using scalar_t    = typename KView::non_const_value_type;
   using exe_space_t = typename KView::execution_space;
   using sum_t       = std::conditional_t<std::is_same<scalar_t, bool>::value, int, scalar_t>;
 
+  // JGF REMOVE
```

Comment:
Is this something that you meant to remove here? Or later?

Reply:
Yeah, I should have removed this.